packages/internal/src/debug.js

```diff
@@ -1,41 +1,108 @@
 // @jessie-check
 
+/**
+ * @import {OptPrefixPath} from './types-internal.js';
+ */
+
+// TODO This is top level static mutable state. Why did it pass at-jessie-check?
 let debugInstance = 1;
 
+const logDisabled = (..._args) => {};
+logDisabled.sub = _ => logDisabled;
```

Comment:
I'm surprised that typechecks. It doesn't pass the typechecker in my brain :)

```suggestion
const logDisabled = Object.assign((..._args) => {}, { sub: _ => logDisabled });
```

Reply:
I do not find the suggested code to be clearer. Ok with you if I do not take this suggestion?

Reply:
as long as I don't get red suqigglies when I try to use `.sub(...)`

Reply:
Good.

I do normally see red squiggles in my vscode for static type errors. So if I don't, I'll guess that you won't either. But I'd appreciate if you check before approving. Thanks.

Reply:
I did end up liking your `Object.assign` pattern better, and using it at #11857 . Thanks. Either way, I still see no read squiggles. Resolving.